pym/portage/dbapi/bintree.py

```diff
@@ -672,17 +693,17 @@ def _populate(self, getbinpkgs=False, getbinpkg_refresh=True):
 					multi_instance = False
 					invalid_name = False
 					build_id = None
-					if myfile.endswith(".xpak"):
+					if myfile.endswith("." + self.suffix):
```

Comment:
I feel like the self.suffix changes should be split into a separate patch. We can choose compression method with having change the suffix code at all, because compression_probe does not care about the file suffix.